pyccel/codegen/printing/fcode.py

```diff
@@ -206,6 +208,20 @@ def __init__(self, parser, prefix_module = None):
 
         self.prefix_module = prefix_module
 
+    def print_constant_imports(self):
+        """Prints the use line for the constant imports used"""
+        macro = "use, intrinsic :: ISO_C_Binding, only : "
+        rename = []
+        for constant in self._constantImports:
+            if isinstance(constant, str):
+                rename.append(constant)
+            else:
+                rename.append(constant[0] + ' => ' + constant[1])
```

Comment:
can you use list comprehension instead of forloop.
```python
rename = [c if isinstance(c, str) else c[0] + ' => ' + c[1] for c in self._constantImports]
```

Reply:
noted!